docker-compose.yaml

```diff
@@ -431,14 +435,16 @@ services:
     image: ghcr.io/espressosystems/espresso-sequencer/sequencer:main
     ports:
       - "$ESPRESSO_SEQUENCER4_API_PORT:$ESPRESSO_SEQUENCER_API_PORT"
-    command: sequencer -- http -- catchup -- status
+    command: sequencer-sqlite -- storage-sql -- http -- query -- catchup -- status
```

Comment:
Shouldn't this work without `sequencer-sqlite` and `storage-sql` if we set `ESPRESSO_SEQUENCER_EMBEDDED_DB=true`?

Reply:
I think we still have to pass storage-sql module as it just loads a different binary?

Reply:
I can update the script to pass the --storage-sql module